source/val/validate_id.cpp

```diff
@@ -2191,14 +2189,12 @@ spv_result_t IdPass(ValidationState_t& _,
         ret = SPV_SUCCESS;
         break;
     }
-    if (SPV_SUCCESS != ret) {
-      return ret;
-    }
-  }
-  if (result_id) {
-    _.RemoveIfForwardDeclared(result_id);
+    if (SPV_SUCCESS != ret) return ret;
   }
-  _.RegisterInstruction(*inst);
+  if (result_id) _.RemoveIfForwardDeclared(result_id);
```

Comment:
I personally go back and forth between braces or no braces around a single short statement.  :-)

Reply:
I prefer no braces, but I really wish we formatted it onto two lines.